recipes-ni/sysconfig-settings/sysconfig-settings.bb

```diff
@@ -155,7 +92,88 @@ pkg_postinst_ontarget:${PN} () {
 
 pkg_prerm_ontarget:${PN} () {
 	rm -f ${systemsettingsdir}/target_common.ini \
```

Comment:
Indentation doesn't match next lines; did you intend to move `${systemsettingsdir}/target_common.ini \` as well to next line like others?

Reply:
The difference in indentation levels is original to the previous version of the file, and is intentional. The latter lines have an additional level of indentation because they are continuations of the `rm ...` command.

Won't fix.